Makefile

```diff
@@ -89,6 +89,9 @@ reports-config-lint: ## lint reports YAML configuration file
 reports-config-rebuild-rst: ## Rebuild sample reports RST docs
 	$(CONFIG_MANAGE) build_rst reports > doc/source/admin/reports_options.rst
 
+pin-linting:
+	sed -i.bak -e "s/>=/==/" tox.ini
```

Comment:
I'm of the idea that we should pin the latest available version on PyPI, which is what we normally use for Travis on the dev branch. This would instead pin the minimum supported version, which we don't update very often (normally only when some version is broken).

Reply:
I thought you might object. This is admittedly a broad stroke - I'd call this a step in the right direction more than a complete answer. I think this is the right process for stable - we just need a separate process to keep dev updated. Something hooked into https://requires.io/ maybe?

What if I add language to the issue to say update dev, commit, and then run this for stable?

Reply:
> which we don't update very often (normally only when some version is broken).

And well it seems to break at least once a release though 😆.